index.html

```diff
@@ -4380,6 +4381,125 @@ <h3>Parsing content</h3>
+      <p>
+       When the user scans a tag, at that point the web application has access
+       to read the data on the tag, and in case it is not read-only, also to
+       write data to the tag. Consumer stickers for private usage (e.g. in the
+       maker community) are often unlocked (read + write), whereas commercial
+       deployment of NFC are read-only.
+      </p>
+      <p class="note">
+        An older protocol SNEP (Simple NDEF Exchange Protocol) allowed active
+        devices (e.g. a phone) to receive NDEF data from another active device,
+        but it is unsupported by Web NFC and it is currently being deprecated
+        on supported native platforms.
+      </p>
+      <p class="note">
+        A newer protocol TNEP (Tag NDEF Exchange Protocol) allows bidirectional
+        communication between a scanner device (e.g. phone) and actively powered
+        device like an IOT device. It is currently unsupported by Web NFC, and
+        furthermore it has restrictions on what input to accept and the IOT device
+        must ensure that the accepted records are valid.
+      </p>
+      <p>
+        If the tag contains privacy sensitive data, such data will be shared with
+        the site. Potentially, not immediately if the UX requires the user to
+        confirm the data exchange before doing so.
+      </p>
+      <p>
+        In some cases it might be obvious that the tag/device contains privacy
+        sensitive data, say in case it is a glucose meter which can indicate
+        that you, or someone in the close family, are a diabetes patient.
+      </p>
+      <p>
+        In other cases it might not be obvious that that can happen, but a user
+        might have used an app or website to write data to a tag that unknowingly
+        to the user encoded a user id or the like, which can later be read back
+        by any other site.
+      </p>
+      <p>
+        Private and unexpected data can also be stored in files (e.g. word
+        pressing documents, PDFs or camera images) which are uploaded using
+        the file upload API. The mitigations associated with the Web NFC API
+        are stronger than those associated with file upload and the data less
+        likely to be personally identifiable.
+      </p>
+    </section>
+    <section> <h4>Reading and writing during a scan</h4>
+      <p>
+        A scan of a tag might also reveal user location if the website knows
+        how to identify the tag and know the tags location in the real world,
+        like if it is mounted inside a museum. It might also be able to deduct
+        it somewhat as for instance FeliCa NFC tags are mostly used in Japan,
+        but Web NFC doesn’t reveal what tag technology is used.
+      </p>
+      <p>
+        This does not bring the web advertising and tracking model into the
+        real world, because it requires an action by the user and it is
+        obvious when being used and cannot be triggered in the background.
```

Comment:
I'm not sure "obvious" is the right word here for end-users.

Reply:
clear?

Reply:
reworded